default.json

```diff
@@ -100,6 +100,16 @@
     {
       "args": ["systemctl", "status", "flanneld.service"],
       "link": "flanneld_status"
+    },
+    {
+      "args": ["slabtop", "-o"]
+    },
+    {
+      "args": ["iptables", "-vnL"],
```

Comment:
If you're going to throw that in you'll also need the same thing with `ip6tables`.  Same with Line 109.